graphql_compiler/scripts/generate_test_sql/utils.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import random
+from uuid import UUID
+
+import six
+
+
+CREATE_VERTEX = 'create vertex '
+CREATE_EDGE = 'create edge '
+SEPARATOR = '__'
+
+
+def get_uuid():
+    """Return a pseudorandom uuid."""
+    return str(UUID(int=random.randint(0, 2**128 - 1)))  # nosec
+
+
+def get_random_net_worth():
+    """Return a pseudorandom uuid."""
+    return int(1e3*random.random() * 100) / 100.0  # nosec
+
+
+def get_random_limbs():
+    """Return a pseudorandom uuid."""
+    return random.randint(2, 10)  # nosec
+
+
+def get_random_choice(choices):
+    """Return a pseudorandom uuid."""
+    return random.choice(choices)  # nosec
+
+
+def get_random_date():
+    """Return a pseudorandom date."""
+    random_year = random.randint(2000, 2018)  # nosec
+    random_month = random.randint(1, 12)  # nosec
+    random_day = random.randint(1, 28)  # nosec
+    return datetime.date(random_year, random_month, random_day)
+
+
+def select_vertex_statement(vertex_type, name):
+    """Return a SQL statement to select a vertex of given type by its `name` field."""
+    template = '(select from {vertex_type} where name = \'{name}\')'
+    args = {'vertex_type': vertex_type, 'name': name}
+    return template.format(**args)
+
+
+def set_statement(field_name, field_value):
+    """Return a SQL clause (used in creating a vertex) to set a field to a value."""
+    if not isinstance(field_name, six.string_types):
+        raise AssertionError(u'Expected string field_name. Received {}'.format(field_name))
+    field_value_representation = repr(field_value)
+    if isinstance(field_value, datetime.date):
```

Comment:
Do `datetime.datetime` objects need similar special handling like this? E.g. another `isoformat()` call etc.